packages/grafana-ui/src/components/Logs/LogDetails.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('LogDetails', () => {
   describe('when labels are present', () => {
     it('should render heading', () => {
       const wrapper = setup(undefined, { labels: { key1: 'label1', key2: 'label2' } });
-      expect(wrapper.find({ 'aria-label': 'Log Labels' })).toHaveLength(1);
+      expect(wrapper.find({ 'aria-label': 'Log Labels' }).hostNodes()).toHaveLength(1);
```

Comment:
Why do we need the `hostNodes()` here?

Reply:
Good question! It is because we are using **mount**.  Therefore wrapper is going to contain React components as well as HTML nodes. So if we add aria-labels or titles to Icon component, both React component and HTML node is going to contain them. Wrapper is going to be something like:
`<Icon aria-label="Log Labels" /> <div aria-label="Log Labels"><svg /></div></Icon>`
More: https://github.com/enzymejs/enzyme/issues/836